src/ape/managers/project.py

```diff
@@ -2592,7 +2601,10 @@ def update_manifest(self, **kwargs):
         self.manifest_path.write_text(manifest_text, encoding="utf8")
 
     def load_contracts(
-        self, *source_ids: Union[str, Path], use_cache: bool = True
+        self,
+        *source_ids: Union[str, Path],
+        use_cache: bool = True,
+        exclude_compiler: Optional[Union[str, list[str]]] = None,
```

Comment:
Simplify typing here
```suggestion
        excluded_compilers: Optional[list[str]] = None,
```